components/Input/input.css

```diff
@@ -0,0 +1,39 @@
+:root {
+    --error-color: rgb(255, 0, 0);
+    --success-color: rgb(0, 128, 0);
+}
+
+.inp-container {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    margin-top: 1rem;
+}
+
+label {
```

Comment:
dont go very generic use a class instead like label--md to have flexibility in the styling

Reply:
ok